src/config.ts

```diff
@@ -85,14 +87,14 @@ const indexRules: Rule[] = [
     objectID: 'promote-exact',
     description: 'promote exact matches',
     condition: {
-      pattern: '{facet:_searchInternal.alternativeNames}',
+      pattern: '{facet:_searchInternal.popularAlternativeNames}',
```

Comment:
we should change the query rule only after the reindex is completed, otherwise this will automatically change everything.

Reply:
True, reverted everything to `alternativeNames` for now.